tests/unit/test_connector.py

```diff
@@ -41,3 +44,5 @@ async def timeout_stub(*args, **kwargs):
         pytest.raises(
             TimeoutError, connector.connect, connect_string, "pymysql", timeout=timeout
         )
+
+    del icm
```

Comment:
I'm not sure I understand why this deletion is necessary

Reply:
Removed this, as well as the other calls to del. There's no need to call it explicitly.